src/components/App.js

```diff
@@ -380,8 +380,15 @@ export default class App extends React.Component {
      *     try another url
      * - If `erroredStreams` has as many items as the list of available streams:
      *   - Pause the player because this means all of our urls are having issues
+     *
+     * Note that this function does not handle internet connection issue (which would also cause player failure).
+     * When the connection dropped, we just return early here.
+     * The OfflineBanner component has some built-in logic that allows it to be displayed/hidden automatically,
+     * and the reconnection is supported by Nchan by default.
      */
 
+    if (!navigator.onLine) return;
```

Comment:
Ignore the connection failure in the error handling function.

The OfflineBanner component has the show/hide logic inside it, which allows it to be displayed/hidden accordingly

https://github.com/freeCodeCamp/coderadio-client/blob/830a540fe99382f4e35b471576528863b88ee00c/src/components/OfflineBanner.js#L6-L14